addon/mixins/in-viewport.js

```diff
@@ -91,14 +91,23 @@ export default Mixin.create({
       return;
     }
 
-    const $contextEl = $(context);
+    const contextBoundingClientRect = context.document.documentElement.getBoundingClientRect();
     const boundingClientRect = element.getBoundingClientRect();
+    let width, height;
+    if (!('width' in contextBoundingClientRect)) {
+      // IE8 and lower:
+      width = contextBoundingClientRect.right - contextBoundingClientRect.left;
+      height = contextBoundingClientRect.bottom - contextBoundingClientRect.top;
```

Comment:
@offirgolan I think this will work to compute the `width` and `height` in IE8. `width` and `height` are in a sense redundant properties on the DOMRect as they can be derived from `right - left` and `bottom - top`, respectively.

Apparently this is quite messy. This article from 2008 by John Resig makes me think its not this simple: http://ejohn.org/blog/getboundingclientrect-is-awesome/

Modern jQuery deals with some of this mess here: https://github.com/jquery/jquery/blob/master/src/offset.js#L78-L118